Dockerfiles/cluster-agent/entrypoint.sh

```diff
@@ -17,7 +17,6 @@ if [[ -z "$DD_DD_URL" ]]; then
     export DD_DD_URL="https://app.datadoghq.com"
 fi
 
-chmod +x /opt/datadog-agent/bin/datadog-cluster-agent/datadog-cluster-agent
 sync	# Fix for 'Text file busy' error
```

Comment:
Do we still need this sync?

Reply:
We had potential race condition issues with the chmod not necessarily over when starting the dca. As I moved it away, I think we can remove the sync.